backend/src/test/java/de/jsmenues/backend/statistic/StatisticControllerTest.java

```diff
@@ -209,10 +215,14 @@ public void createChartWithInputStreamAndChartName() {
         multiPart.bodyPart(new FormDataBodyPart("file", String.valueOf(new ByteArrayInputStream(new byte[0]))));
         multiPart.setMediaType(MediaType.MULTIPART_FORM_DATA_TYPE);
         Response mockResonse = Response.ok().build();
-        when(service.createChart(anyString(), anyString(), anyString(), any(InputStream.class), any(FormDataContentDisposition.class))).thenReturn(mockResonse);
+        /*
+         * the expected "groupName" and "description" parameters in the API call "/createChart" are not defined in the FormData of this test,
+         * thus they will be received as null values. Since Mockito 2.1.0 anyString() can no longer be a null value. Thus nullable(String.class) has to be used.
+         */
+        when(service.createChart(anyString(), nullable(String.class), nullable(String.class), any(InputStream.class), any(FormDataContentDisposition.class))).thenReturn(mockResonse);
```

Comment:
Same thing here as in the other test.
Replace the wildcards and nullables for the important parameters chartname, groupname and description. Then please remove this comment